setup/plugins/get_plugins.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
"code" type files, like scripts, should include the standard pi-stomp header (see setup/audio/audiocard-setup.sh)

Reply:
added license